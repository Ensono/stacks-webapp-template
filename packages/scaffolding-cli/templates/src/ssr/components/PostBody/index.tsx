import {documentToReactComponents} from "@contentful/rich-text-react-renderer"
import React from "react"

export default function PostBody({content}) {
    return (
        <div>
            <div>{documentToReactComponents(content)}</div>
        </div>
    )
}
