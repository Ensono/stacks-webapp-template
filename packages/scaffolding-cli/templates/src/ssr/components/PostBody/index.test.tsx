import {render} from "@testing-library/react"
import React from "react"
import PostBody from "."

test("renders ApiPane snapshot", () => {
    const {asFragment} = render(<PostBody content="" />)
    expect(asFragment()).toMatchSnapshot()
})
