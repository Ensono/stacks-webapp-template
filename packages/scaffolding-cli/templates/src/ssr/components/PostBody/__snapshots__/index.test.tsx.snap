// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ApiPane snapshot 1`] = `
<DocumentFragment>
  <div>
    <div />
  </div>
</DocumentFragment>
`;
